README.md

```diff
@@ -31,11 +36,106 @@ To start local dev environment with HMR support:
 
 ⚠️ `test:e2e:ci` command should launch tests on the generated build production. For now, it launches the vite dev server, otherwise the code coverage report is not generated and the next ci command `npx nyc report` will trigger an error.
+
+## 📦 Build as a lib for integration (in geoportal v3)
+
+In order to use the new Lux components made with Vuejs as an external dep, follow steps below.
+
+### Build the lib
+
+In the project **luxembourg-geoportail**, build the app as library with this command:
+
+```
+$ npm run build:lib:prod
+```
+
+When using the lib in local developpement mode within the geoportailv3 docker composition, use (this sets the necessary base URL option `--base=/dev/main`):
+
+```
+$ npm run build:lib:dev
+```
+
+This will generate an export of all available **Vue components** and **State persistor services** that can be used in a independant application (eg. geoportalv3 legacy app).
+
+To see what are the components exported, check the `/src/bundle/lib.ts` that is the entry point for the build of the library.
+
+⚠️ This build includes its own **Vue.js** dependencies and will export Vue's utilities such as watchers that can be used to interact with the lib state. For this reason, avoid including another Vue.js dep in the targeted project.
+
+### Import the lib in another app
+
+You can include the built lib multiple ways in the `package.json`:
+
+- using `npm link`
+- using `git`
+
+```js
+// Example of a package.json
+// including the lib with git and a specific branch (GSLUX-602-IntegrationV3)
+"packages": {
+    "": {
+      "name": "luxgeo_test_inte",
+      "version": "1.0.0",
+      "license": "ISC",
+      "dependencies": {
+        "luxembourg-geoportail": "git://github.com/Geoportail-Luxembourg/luxembourg-geoportail.git#GSLUX-602-IntegrationV3"
+        // or "https://github.com/Geoportail-Luxembourg/luxembourg-geoportail.git#GSLUX-602-IntegrationV3"
+      },
+      "devDependencies": {
+        // ...
+      }
+    }
+}
+```
+
+### Develop on lib within geoportailv3 environnement
+
+A simple way to develop on the lib and test it directly from within the geoportailv3 context is to map your `luxembourg-geoportal` repository as a volume to webpack_dev_server service of the docker composition:
+
+```
+  webpack_dev_server:
+    volumes:
+      # map vue custom elements for dev:
+      - /[your-path]/luxembourg-geoportail:/usr/lib/node_modules/luxembourg-geoportail
+```
+
+To see changes applied you then need to run `npm run build:lib:dev` and refresh your browser on `http://localhost:8080/dev/main.html`.
+
+### Make assets available
+
+Assets like translations are imported using an absolute path. Make them accessible via the right path. For geoportalv3, we choose to copy the assets directory with the `CopyPlugin`:
```

Comment:
Not sure if we should get rid of this section. Finally we did this in the [Dockerfile](https://github.com/Geoportail-Luxembourg/geoportailv3/blob/integration-vue/geoportal/Dockerfile#L61).